src/validation.cpp

```diff
@@ -5729,7 +5743,7 @@ bool ChainstateManager::ActivateSnapshot(
                     "Manually remove it before restarting.\n"), fs::PathToString(*snapshot_datadir)));
             }
         }
-        return false;
+        return util::Error{_(reason)};
```

Comment:
re: https://github.com/bitcoin/bitcoin/pull/30267#discussion_r1663631973

> It is not possible to pass a raw c-string pointer to transifex for translation. Only string literals can be translated.

In case it's not clear, the fix for this problem and the previous [unnecessary translation](https://github.com/bitcoin/bitcoin/pull/30267#discussion_r1663611194) problem is probably to replace `_()` with `Untranslated()` everywhere where in this PR when converting strings to bilingual_str. If we added a snapshot loading GUI in the future, it might make sense to add translations at that point.

Reply:
re: https://github.com/bitcoin/bitcoin/pull/30267#discussion_r1663631973

> It is not possible to pass a raw c-string pointer to transifex for translation. Only string literals can be translated.

Fixed in #30417